src/components/generics/Markdown.vue

```diff
@@ -222,12 +222,16 @@ figure[c2c\:role='embedded-figure'] {
 
 // medium image
 figure[c2c\:size='MI'] {
-  width: 25vw;
   max-width: 400px;
```

Comment:
only IE11 in our current targeted browsers do not support `min()`. This compatibility layer should be removed when support is dropped. I added the mention in #1131